include/bl_storage.rst

```diff
@@ -0,0 +1,32 @@
+.. _doc_bl_storage:
+
+Bootloader storage
+##################
+
+The bootloader storage library is used by the :ref:`bootloader` to read from and write to a protected data area that contains the provisioned bootloader data.
+The library can also be included into the application to allow reading the provisioned data.
+
+The provisioned data is stored in the one-time programmable (OTP) region of UICR, if available.
```

Comment:
Maybe note the devices here 
nRF9160, nRF5340 APP for OTP, otherwise write protect

Reply:
I actually prefer the current way, as it is easier to maintain. When more chips arrive with OTP we don't have to remember to go back here and update the docs to keep it up to date.

It is a bit more cumbersome for the user, but IMO it is more sustainable. Alternatively we could list the devices as examples "for example nRF9160 and nRF5340 APP".

Reply:
Added something like @hakonfam's suggestion in parentheses, take a look.